tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -62,6 +62,29 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
         }
     }
 
+    const auto& process_name = shared::GetCurrentProcessName();
+    Logger::Info("ProcessName: ", process_name);
+
+    const auto& process_command_line = shared::GetCurrentProcessCommandLine();
+    Logger::Info("Process CommandLine: ", process_command_line);
+
+    // CI visibility checks
+    if (!process_command_line.empty())
+    {
+        bool is_ci_visibility_enabled = false;
+        if (shared::TryParseBooleanEnvironmentValue(shared::GetEnvironmentValue(environment::ci_visibility_enabled),
+                                                    is_ci_visibility_enabled) &&
+            is_ci_visibility_enabled)
+        {
+            if (process_name == WStr("dotnet") && process_command_line.find(WStr("testhost.dll")) == WSTRING::npos)
```

Comment:
What about other testshosts?
We tested for this `BeginsWith(GetExeName(), "testhost"_W))`, it could be safer for Net FW

Reply:
as we talked offline, this is a different comparison, here I'm not comparing the process name but the argument of the process...